utils/src/main/java/org/robolectric/util/Util.java

```diff
@@ -99,11 +99,11 @@ public static Path pathFrom(URL localArtifactUrl) {
   }
 
   public static List<Integer> intArrayToList(int[] ints) {
```

Comment:
nit: can this just be removed? It's not used anywhere in the code.

Reply:
We can mark it deprecated, and remove it at next release, because it is public to users.

Reply:
I will send a new PR to mark it deprecated this weekend.